pkg/controller/kubelet-config/kubelet_config_controller.go

```diff
@@ -507,6 +531,16 @@ func (ctrl *Controller) syncKubeletConfig(key string) error {
 		return ctrl.syncStatusOnly(cfg, err)
 	}
 
+	// Fetch the NodeConfig
+	nodeConfig, err := ctrl.nodeConfigLister.Get(ctrlcommon.ClusterNodeInstanceName)
+	if macherrors.IsNotFound(err) {
+		nodeConfig = createNewDefaultNodeconfig()
+	} else if err != nil {
+		glog.V(2).Infof("%v", err)
```

Comment:
V(2) is the default verbosity so this is always logged. Do you need to do that given that you are returning the error right below?

Reply:
Addressed